### ISSUE TITLE
Stroke all masks sequentially not working

### ISSUE THREAD

**Issue by MauriceArikoglu** [airbnb/lottie-web#2684]
**Tell us about your environment**
Mac

* **Browser and Browser Version:**
* **After Effects Version:**
Firefox 89
AE 22.2

**What did you do? Please explain the steps you took before you encountered the problem.**
Applying a stroke effect to a masked glyph with >1 masks, the strokeAllMasks is true and set to stroke sequentially. Timing is correct in AE, animation starts at the same time in Lottie-Web (svg renderer)

**What did you expect to happen?**
Animation to play sequentially 

**What actually happened? Please include as much _relevant_ detail as possible.**
See above

**Please provide a download link to the After Effects file that demonstrates the problem.**
On mobile Right now

**Comment by bodymovin** [comment]
yes, unfortunately the stroke mask effect doesn't support sequential strokes yet

**Comment by MauriceArikoglu** [comment]
@bodymovin thanks for the heads-up, can I offer my help for integration? Or are you aware of any alternative solutions to enable this kind of effect?
